app/assets/stylesheets/components/_glance-metric.scss

```diff
@@ -0,0 +1,53 @@
+@import "../../../node_modules/govuk-frontend/helpers/colour";
+@import "../../../node_modules/govuk-frontend/helpers/visually-hidden";
+
+.app-c-glance-metric {
+  border-top: 2px solid $govuk-border-colour;
+  border-bottom: 1px solid $govuk-border-colour;
+
+  @include media(mobile) {
+    border-bottom: 0;
+  }
+}
+
+.app-c-glance-metric__heading {
+  @include govuk-responsive-margin(3, "top");
+  @include govuk-responsive-margin(2, "bottom");
+
+  @include media-down(tablet) {
+    min-height: 4em;
+  }
+
+  @include media-down(mobile) {
+    min-height: 0;
+  }
+}
+
+.app-c-glance-metric__context {
+  min-height: 5em;
+  border-bottom: 1px solid $govuk-border-colour;
+
+  @include media(mobile) {
+    min-height: 0;
+  }
+}
+
+.app-c-glance-metric__link-wrapper {
+  min-height: 1.4em;
+
+  @include media-down(tablet) {
+    min-height: 2.8em;
+  }
+
+  @include media(mobile) {
+    min-height: 0;
+  }
+}
+
+.app-c-glance-metric__trend__text {
```

Comment:
According to BEM this should be `__trend-text` as it doesn't allow chaining elements.